templates/repo/commits.tmpl

```diff
@@ -5,14 +5,12 @@
 	  <div class="ui secondary menu">
 	    {{template "repo/branch_dropdown" .}}
 	    <div class="fitted item">
-	      <div class="ui basic small button">
```

Comment:
I think I removed both div tags and moved class attribute from div to a. Am I missing something? I did not do formatting so the change stays clear.